src/main/java/duke/command/CommandType.java

```diff
@@ -0,0 +1,8 @@
+package duke.command;
+
+/**
+ * Contains the different types of commands available in duke.
+ */
+public enum CommandType {
```

Comment:
abstracting out command types is a good idea! :) this helps to reduce errors in parsing and extending the app's functionality.